src/utils.tsx

```diff
@@ -586,3 +586,5 @@ export const deselectRowById = <TData extends MRT_RowData>(
     return updated;
   });
 };
+
+export const booleanSelectFilterOptions = ['Yes', 'No'];
```

Comment:
As its a constant it should probably be all caps, might be worth moving next to the others above in the file and matching their naming e.g. `COLUMN_FILTER_BOOLEAN_OPTIONS` or something.